src/endpoints/catalogs.js

```diff
@@ -29,8 +29,15 @@ class Nodes {
   }
 
   GetNodeChildren({ nodeId, token = null }) {
+    const { limit, offset, includes, sort, filter } = this
     return this.request.send(
-      `catalogs/${this.endpoint}/${nodeId}/relationships/children`,
+      buildURL(`catalogs/${this.endpoint}/${nodeId}/relationships/children`, {
+        includes,
+        sort,
+        limit,
+        offset,
+        filter
```

Comment:
@psin32 are `includes`, `sort`, `filter` available on these endpoints?

Reply:
@samblacklock I have removed `includes` and `filter` from this.